stratum/values.yaml

```diff
@@ -9,7 +9,14 @@ image:
   #     --docker-server=registry.aetherproject.org --docker-username=<DOCKER_USER> --docker-password=<DOCKER_PASS>
   # pullSecrets:
   #   - aether-registry-credential
- 
+
+annotations:
+  fluentbit.io/parser: stratum
+
+extraParams:
+  - "-log_dir=/dev/null"
+  - "-v=0"
+  - "-stderrthreshold=0"
```

Comment:
Nitpick: for consistency I'd use either the longform `--flag` or the short form `-flag`. Both are supported, short is preferred.

Reply:
Just to clarify, you want to use the short form for `-bf-sim`, `-chassis_config_file`, and all the `extraParams`?

Reply:
I'd prefer that, yes. Their all Stratum flags, but the different forms might lead somebody to think they are not.